packages/firestore/test/unit/specs/existence_filter_spec.test.ts

```diff
@@ -631,4 +631,189 @@ describeSpec('Existence Filters:', [], () => {
         .expectLimboDocs(...docKeys.slice(50))
     );
   });
+
+  specTest('Bloom filter handles new document adds', [], () => {
+    const query1 = query('collection');
+    const docs = [];
+    for (let i = 0; i < 100; i++) {
+      docs.push(doc(`collection/doc${i}`, 1000, { v: 1 }));
+    }
+    const docKeys = docs.map(item => item.key);
+
+    const bloomFilterProto = generateBloomFilterProto({
+      contains: docs.slice(0, 50),
+      notContains: docs.slice(50),
+      bitCount: 1000,
+      hashCount: 16
+    });
+    return (
+      spec()
+        .userListens(query1)
+        .watchAcksFull(query1, 1000, ...docs)
+        .expectEvents(query1, { added: docs })
+        // Doc0 to doc49 are deleted in the next sync.
+        .watchFilters([query1], docKeys.slice(0, 50), bloomFilterProto)
+        .watchSnapshots(2000)
+        // BloomFilter correctly identifies docs that deleted, skip full query.
+        .expectEvents(query1, { fromCache: true })
+        .expectLimboDocs(...docKeys.slice(50))
+    );
+  });
+
+  specTest('Existence filter when there is no changes', [], () => {
```

Comment:
nit: update the names of these new test cases to mention "bloom filter" somehow. Otherwise, they just sound like duplicates of other test cases that undoubtedly exist to verify existence filter behavior.